nginx-php-fpm-apb/README.md

```diff
@@ -0,0 +1,19 @@
+nginx-php-fpm-apb
+======================
+
+An apb for deploying nginx and php-fpm containers.
+
+## What it does
+* Deploys 1 nginx container and 1 php-fpm container and links them.
+
+## Requirements
+* N/A
+
+## Parameters
+* N/A
+
+## Running the application
+`docker run -e "OPENSHIFT_TARGET=<openshift_target>" -e "OPENSHIFT_USER=<user>" -e "OPENSHIFT_PASS=<password>" ansibleplaybookbundle/nginx-php-fpm-apb provision`
```

Comment:
Small nit: This actually would not work. The correct command here is:
`docker run -e "OPENSHIFT_TARGET=<openshift_target>"  -e "OPENSHIFT_TOKEN"=<openshift_token>" ansibleplaybookbundle/nginx-php-fpm-apb provision`

I apologize if you got this command from another stale doc in our repo, this recently changed and I have tried to make an effort to update them.